int32 home_score
int32 away_score
int32 home_bot_count
int32 away_bot_count
int32 remaining_seconds
bool play
bool reset_field
bool second_half